### PR TITLE
Fix 'frontend' currently not working

### LINKED ISSUE #601: Docker build for 'frontend' currently not working because npm is outdated
# Bug Report

The OpenTelemetry demo with Docker does not work because npm is outdated in the base image. I used the current build v1.1.0.

## Symptom

The OTel demo app won't start, and I can't access the frontend interface because the container keeps restarting due to an outdated npm dependency.

```
frontend                 | npm notice 
frontend                 | npm notice New major version of npm available! 8.19.2 -> 9.1.2
frontend                 | npm notice Changelog: <https://github.com/npm/cli/releases/tag/v9.1.2>
frontend                 | npm notice Run `npm install -g npm@9.1.2` to update!
frontend                 | npm notice 
frontend                 | npm ERR! path /app
frontend                 | npm ERR! command failed
frontend                 | npm ERR! signal SIGKILL
frontend                 | npm ERR! command sh -c -- node --require ./Instrumentation.js server.js
frontend                 | 
frontend                 | npm ERR! A complete log of this run can be found in:
frontend                 | npm ERR!     /home/nextjs/.npm/_logs/2022-11-21T14_37_59_716Z-debug-0.log
```

## Reproduce
1. Download the repo (`$git clone`)
2. Try to startup with Docker (`$docker compose up`)

## Additional Context

I used an M1 mac, but I also rebuilt all docker dependencies with the `$docker compose` command as described in the documentation.

### PR DESCRIPTION
**Pull request by tommyziegler**
Fixed the alpine base image as the npm version has a newer major version. See topic: https://github.com/open-telemetry/opentelemetry-demo/issues/601

Fixes #601.

## Changes

Fixed alpine base image, to make npm dependency for major version 9.x again work.

### CODE REVIEW

File: src/frontend/Dockerfile [hunk 1]
```diff
@@ -1,12 +1,12 @@
-FROM node:16-alpine AS deps
+FROM node:19-alpine AS deps
```
Comment (julianocosta89) [inline src/frontend/Dockerfile:1]: Please use the current LTS version: https://nodejs.org/en/

**Review by julianocosta89 — CHANGES_REQUESTED** [review]

**Comment by julianocosta89** [comment]
@tommyziegler could you take a look at the comment https://github.com/open-telemetry/opentelemetry-demo/pull/605#discussion_r1028437017?

File: src/frontend/Dockerfile (hunk 1 above)
Comment (tommyziegler) [reply src/frontend/Dockerfile]: So you mean?

`FROM node:19.1-alpine AS deps`

Comment (julianocosta89) [reply src/frontend/Dockerfile]: @tommyziegler nope, I meant the LTS version:

>Active LTS - New features, bug fixes, and updates that have been audited by the LTS team and have been determined to be appropriate and stable for the release line.

That would be: 
`FROM node:18-alpine`

Comment (tommyziegler) [reply src/frontend/Dockerfile]: Ok, have done it now ... Thx. Have a nice weekend

**Review by julianocosta89 — APPROVED** [review]
Thanks @tommyziegler!

**Review by puckpuck — APPROVED** [review]

**Pull request merged**